### DISCUSSION TITLE
Using Premake branding

### DISCUSSION THREAD

**Discussion by colton-mcgraw** [premake/premake-core#2380] [Q&A] [answered]
Hello,

I have created a Visual Studio Extension to allow you to run premake5 from within Visual Studio by pressing a button.
I would like to name it VSPremake and use the premake icon as the button icons.

Would this be infringing on the Premake brand? If so who could I ask for permission to use the branding?

I have include some screen grabs of what the extension looks like.

<img width="256" alt="image" src="https://github.com/user-attachments/assets/856b8b6e-b8dc-4e95-a0cb-9cd3ded831ca" />
<img width="341" alt="image" src="https://github.com/user-attachments/assets/79b2503d-b8f3-442e-9914-b7dad1006cb3" />
<img width="826" alt="image" src="https://github.com/user-attachments/assets/3792d4e7-ff7c-449e-9407-f66c7fbb445f" />
<img width="287" alt="image" src="https://github.com/user-attachments/assets/1a93b854-fb8f-48d0-8c85-1c199064b137" />

**Comment by nickclark2016** [comment]
I don't know the answer on this, but I'll get someone who does to take a look.

**Comment by nickclark2016** [comment, marked as answer]
Okay, so I think the official stance here is "Don't market yourself as official Premake", and link back to `https://premake.github.io` in the description of the plugin, then you should be good to go. Does this seem reasonable? @ColtMcG1

**Reply by colton-mcgraw** [reply]
Thank you very much! I will do this. If there is anything further that needs to be changed, please reach out!
